Nuqleon/Core/LINQ/Nuqleon.Linq.Expressions.Optimizers/System/Linq/Expressions/PureMemberCatalog.cs

```diff
@@ -742,7 +742,7 @@ public static class System
                 (char character) => global::System.Uri.IsHexDigit(character),
                 (string pattern, int index) => global::System.Uri.IsHexEncoding(pattern, index),
                 (string stringToEscape) => global::System.Uri.EscapeDataString(stringToEscape),
-                (string stringToEscape) => global::System.Uri.EscapeUriString(stringToEscape),
+                //(string stringToEscape) => global::System.Uri.EscapeUriString(stringToEscape),
```

Comment:
So I misread the diff—for some reason it looked to me like this was a newly-added commented-out line, rather than commenting out an existing one.

I would say we remove it, on the grounds that nobody should be using it, and we've not yet done a non-preview release, so this is an excellent opportunity to ensure that we never supported it.